<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#EEF;</string>
  </property>
  <widget class="QWidget" name="widget_central">
   <widget class="QLabel" name="lbl_logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res/images.qrc">:/logos/logo.svg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_chronovise">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>chronovise-gui</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pc_about">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>About...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gb_configuration">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>451</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuration</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_iid_test">
       <property name="text">
        <string>EVT i.i.d. test:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cb_iid_test">
       <item>
        <property name="text">
         <string>PPI</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Custom tests</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_custom_test_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>167777</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>↳Test 1:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cb_custom_test_1"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_custom_test_2">
       <property name="text">
        <string>↳Test 2:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cb_custom_test_2"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_custom_test_3">
       <property name="text">
        <string>↳Test 3:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cb_custom_test_3"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbl_EVT_approach">
       <property name="text">
        <string>EVT Approach:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lbl_EVT_estimator">
       <property name="text">
        <string>EVT Estimator:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lbl_GoF_test">
       <property name="text">
        <string>GoF Test:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="cb_EVT_approach">
       <item>
        <property name="text">
         <string>Block-Maxima</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Peak-over-Threshold</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MBPTA-CV</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="cb_EVT_estimator">
       <item>
        <property name="text">
         <string>MLE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PWM</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="cb_GoF_test">
       <item>
        <property name="text">
         <string>Kolmogorov-Smirnov</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Anderson-Darling</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Modified Anderson-Darling</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pb_reset">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_compute">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Compute</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gb_results">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>451</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Results</string>
    </property>
    <widget class="QLabel" name="lbl_ppi_test">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>58</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>PPI Test:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_stat">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>71</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Statistics</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_stat_test_ppi">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>0.005</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_res_test_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>44</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Test 1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_res_test_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>44</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Test 2: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_cv">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Critical Value</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_output">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_res_test_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>44</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Test 3: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_cv_test_ppi">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>0.005</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_stat_test_1">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>0.005</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_stat_test_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>0.005</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_stat_test_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>0.005</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_cv_test_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>150</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>0.005</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_cv_test_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>0.005</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_cv_test_1">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>0.005</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_out_test_ppi">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>60</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red</string>
     </property>
     <property name="text">
      <string>FAIL</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_out_test_1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>90</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:green</string>
     </property>
     <property name="text">
      <string>PASS</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_out_test_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>120</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red</string>
     </property>
     <property name="text">
      <string>FAIL</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_out_test_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>150</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red</string>
     </property>
     <property name="text">
      <string>FAIL</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_res_pwcet_lbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>141</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Estimated pWCET:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_res_pwcet">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>221</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>GEV(100, 0.12, -0.3)</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_res_pwcet_103_lbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>141</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>pWCET at 10^(-3):</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_res_pwcet_109_lbl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>141</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>pWCET at 10^(-9):</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_res_pwcet_103">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>240</y>
       <width>221</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>190</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_res_pwcet_109">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>270</y>
       <width>221</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>260</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_plot_config">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>440</y>
      <width>521</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Plot configuration</string>
    </property>
    <widget class="QLabel" name="lbl_function_to_plot">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>121</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Function to plot:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_pdf">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>109</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_cdf">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>109</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>CDF</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_ccdf">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>109</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>CCDF</string>
     </property>
    </widget>
    <widget class="QLabel" name="lvl_y_scale">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>121</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>y-axis scale:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_linear">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Linear</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_log">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>80</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Logarithmic</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_x_axis">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>121</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>x-axis:</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_x_scale">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>150</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_x_scale">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>81</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>scale:</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_x_pos">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>190</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_x_pos">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>81</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>position:</string>
     </property>
    </widget>
   </widget>
   <widget class="QwtPlot" name="qwtPlot">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>119</y>
      <width>531</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_open_input">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Input File...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>420</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>WCET</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>90</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>p</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res/images.qrc"/>
 </resources>
 <connections/>
</ui>
